fontforge/autowidth.c

```diff
@@ -1086,7 +1086,7 @@ int AW_ReadKernPairFile(char *fn,WidthInfo *wi) {
     unichar_t buffer[300];
     struct kernsets ks;
 
-    filename = utf82def_copy(fn);
+    filename = utf82fsys_copy(fn);
```

Comment:
There are only about 8 comments in all of `ucharmap.c`. What is the difference between these two functions?